src/WebComponents/build/boot-lite.js

```diff
@@ -37,14 +37,12 @@
       }
     }
     // log flags
+    flags.log = {};
```

Comment:
You're right, sorry for the inconvenience and thank you for taking the time for review this.

I'm going to close this PR if you don't mind.

Thanks.